src/main/java/frc/robot/Robot.java

```diff
@@ -11,54 +11,85 @@ public class Robot extends TimedRobot {
     public Drivetrain drivetrain;
     public Auto auto;
     public Arm arm;
-    public DataSender dataSender;
+    //public DataSender dataSender;
     public LED led;
     public Claw claw;
     public Dashboard dashboard;
 
+    private int i;
```

Comment:
single-letter variable names are fine for little things like loop counters but not at all helpful for properties of a class